src/random-seed/random-seed.c

```
@@ -122,6 +126,39 @@ static int parse_argv(int argc, char *argv[]) {
         return 1;
 }
 
+static void write_entropy_xattr(int fd, int entropy_count)
+{
+        char buf[32];
+        int len;
+
+        len = snprintf(buf, sizeof(buf), "%d", entropy_count);
+        if (fsetxattr(fd, XATTR_NAME, buf, len+1, 0)) {
+                if (errno != ENOTSUP)
+                        log_warning_errno(errno, "Failed to set xattr '%s'", XATTR_NAME);
```

please output the error, i.e. add ": %m" to the format string, and also say this error is ignored

---

and it think this log msg should be generate in all cases, however at different log levels. Hence please use log_full_errno() and as first argument pass `errno == EOPNOTSUPP ? LOG_DEBUG : LOG_WARNING`.